src/seed.ts

```diff
@@ -21,10 +32,12 @@ async function seed(): Promise<void> {
       store.description = faker.company.catchPhrase();
       store.vatnumber = vatnumber();
       store.postcode = faker.random.number(9999).toString();
+      store.tags = Promise.resolve([tags[Math.floor(Math.random() * tags.length)]]);
```

Comment:
Aha, nu heb ik door waarom die `Promise.resolve` nodig is.

Met faker kun je ook iets gemakkelijker random integers maken: `faker.random.number(tags.length)`.